src/data/faq_de.json

```diff
@@ -1887,6 +1887,67 @@
+                        ]
+                    },
+                    {
+                        "title": "Wie kommt es zu Unterschieden in der Risikobewertung auf mehreren Geräten, auch wenn die jeweiligen Smartphones sich nicht weit voneinander entfernt befanden?",
+                        "anchor": "risk_encounter_different_devices",
+                        "active": false,
+                        "textblock": [
+                            "Abweichende Risikobewertungen können mehrere Ursachen haben, beeinträchtigen aber nicht die Wirksamkeit der Corona-Warn-App.",
+                            "Die Einschätzung eines möglichen Infektionsrisikos hängt von ermittelter Kontaktdauer und Abstand zum Smartphone eines anderen CWA-Nutzers ab, der sich dazu entschlossen hat, sein positives Testergebnis zu teilen. Die Ermittlung des Abstandes wird dabei auch von der Ausrichtung der beiden Smartphone-Antennen zueinander beeinflusst. Daher ist das Ergebnis der Risikoberechnung auch abhängig von der genauen räumlichen Lage und Ausrichtung der beteiligten Smartphones. Auch die Umgebung (reflektierende oder absorbierende Oberflächen) können die Reichweite des BLE-Signals beeinflussen, sodass je nach Ausrichtung der Smartphones unterschiedliche Werte für Kontaktdauer und Signaldämpfung (Abstand) empfangen werden können.",
+                            "Zudem laufen viele Prozesse auf den Smartphones im Hintergrund ab und können daher zu unterschiedlichen Zeitpunkten starten. Beispiele für solche Prozesse sind die Risikoberechnung und das bei bestehendem WiFi-Zugang mehrfach am Tag stattfindende Laden von Positivschlüsseln. Aber auch die Suche nach anderen Smartphones mit aktiver CWA ist ein solcher Hintergrundprozess.",
+                            "Des Weiteren unterstützt die CWA (abhängig von der Versionsnummer der App und dem genutzten Betriebssystem) unterschiedliche Versionen des Exposure Notification Framework (ENF). Dieses wird von Google und Apple entwickelt und zur Verfügung gestellt. Da nicht jedes Smartphone die neueste Version des ENF unterstützt, können unterschiedliche ENF Versionen zu abweichenden Risikoberechnungen führen.",

```

Comment:
I don't believe that this applies to the Google Exposure Notification System and I have not seen any such restriction documented by Google.

This is probably just an iOS issue.

Reply:
Due to the case mentioned by Tim and possible further cases, of which nothing is known so far or which will still occur in the future, we are of the opinion that it is appropriate to leave the general sentence as it is.